test/lib/actions/RulesAction.test.ts

```diff
@@ -1,34 +1,39 @@
-import * as path from 'node:path';
-import * as fsp from 'node:fs/promises';
 import ansis from 'ansis';
-import {RulesAction, RulesDependencies, RulesInput} from '../../../src/lib/actions/RulesAction';
-import {RulesActionSummaryViewer} from '../../../src/lib/viewers/ActionSummaryViewer';
-import {StubDefaultConfigFactory} from '../../stubs/StubCodeAnalyzerConfigFactories';
+import * as fsp from 'node:fs/promises';
+import * as path from 'node:path';
+import { RulesAction, RulesDependencies, RulesInput } from '../../../src/lib/actions/RulesAction';
+import { RulesActionSummaryViewer } from '../../../src/lib/viewers/ActionSummaryViewer';
+import { DisplayEventType, SpyDisplay } from '../../stubs/SpyDisplay';
+import { SpyRuleViewer } from '../../stubs/SpyRuleViewer';
+import { SpyRuleWriter } from '../../stubs/SpyRuleWriter';
+import { StubDefaultConfigFactory } from '../../stubs/StubCodeAnalyzerConfigFactories';
 import * as StubEnginePluginFactories from '../../stubs/StubEnginePluginsFactories';
-import {SpyRuleViewer} from '../../stubs/SpyRuleViewer';
-import {DisplayEventType, SpyDisplay} from '../../stubs/SpyDisplay';
 
 const PATH_TO_GOLDFILES = path.join(__dirname, '..', '..', 'fixtures', 'comparison-files', 'lib', 'actions', 'RulesAction.test.ts');
 
 describe('RulesAction tests', () => {
 	let viewer: SpyRuleViewer;
+	let spyDisplay: SpyDisplay;
+	let actionSummaryViewer: RulesActionSummaryViewer;
+	let defaultDependencies: RulesDependencies;
```

Comment:
Most of the changes in here are from tidying and bringing things into the `beforeEach` loop.